spec/teaspoon_env.rb

```diff
@@ -12,7 +12,7 @@
 
   # Paths that will be appended to the Rails assets paths
   # Note: Relative to `config.root`.
-  config.asset_paths = ['spec/javascripts', 'spec/javascripts/stylesheets']
+  # config.asset_paths = ['spec/javascripts', 'spec/javascripts/stylesheets']
```

Comment:
Again, let's just remove comments like this. They just turn in to lint over time.